### PR TITLE
feat: Change endpoint names to use nodes instead of components

### PR DESCRIPTION
**Pull request by andrewelamb**
Depends on https://github.com/Sage-Bionetworks/sage-monorepo/pull/1918
fixes [fds-908](https://sagebionetworks.jira.com/browse/FDS-908)

### CODE REVIEW

File: apps/schematic/api/schematic_api/controllers/schema_controller_impl.py [hunk 1]
```diff
@@ -1,42 +1,91 @@
...
+from schematic_api.models.validation_rules_page import ValidationRulesPage
+from schematic_api.models.validation_rule import ValidationRule
+from schematic_api.models.nodes_page import NodesPage
+from schematic_api.models.node import Node
 from schematic_api.controllers.utils import handle_exceptions
 
 
-def get_component_attributes(
-    component_label: str,
+def get_node_label_from_schematic(
```
[the remark this answers is not in the document]
Comment (mialy-defelice) [reply apps/schematic/api/schematic_api/controllers/schema_controller_impl.py:16]: There is actually another function in schematic that would figure out the node label, so this naming would probably make someone think thats what was happening here.

File: apps/schematic/api/schematic_api/controllers/schema_controller_impl.py [hunk 2]
```diff
@@ -1,42 +1,91 @@
...
     schema_explorer = SchemaExplorer()
     schema_explorer.load_schema(schema_url)
-    return schema_explorer.find_class_specific_properties(component_label)
+    return schema_explorer.find_class_specific_properties(node_label)
 
 
 @handle_exceptions
-def list_component_attributes(
-    component_label: str,
+def list_node_attributes(
```
Comment (mialy-defelice) [inline apps/schematic/api/schematic_api/controllers/schema_controller_impl.py:80]: Same comment as above this is getting the properties associate with the node or attribute.

File: apps/schematic/api/schematic_api/openapi/openapi.yaml [hunk 3]
```diff
@@ -232,18 +232,106 @@ paths:
       tags:
       - Storage
       x-openapi-router-controller: schematic_api.controllers.storage_controller
-  /components/{componentLabel}/attributes:
+  /nodes/{nodeDisplay}/nodeLabel:
```
Comment (mialy-defelice) [inline apps/schematic/api/schematic_api/openapi/openapi.yaml:235]: Please refer to my comments in the function calls and apply changes to this file as well.

**Review by mialy-defelice — CHANGES_REQUESTED** [review]

**Comment by andrewelamb** [comment]
@mialy-defelice Could you re-review the changes I just made?

**Review by mialy-defelice — APPROVED** [review]

**Pull request merged**